app/views/admin/agents/plage_ouvertures/index.json.jbuilder

```diff
@@ -12,7 +12,7 @@ json.array! @plage_ouverture_occurrences do |plage_ouverture, occurrence|
   json.rendering "background"
   json.extendedProps do
     json.organisationName plage_ouverture.organisation.name
-    json.location plage_ouverture.lieu.address
-    json.lieu plage_ouverture.lieu.name
+    json.location plage_ouverture.lieu&.address&.presence || "N/A"
+    json.lieu plage_ouverture.lieu&.name&.presence || "N/A"
```

Comment:
`N/A` c'est l'abbréviation de `Not Applicable`, qu'on a dans certains champs vide parce que c'est ce que propose par défaut papertrail (je crois), mais est-ce que ça vaut vraiment le coup de se mettre à avoir ça ailleurs ? Dans le cadre d'une api j'ai l'impression qu'un `null` aurait plus de sens, non ?